### PR TITLE
tvnamer: init at 2.3

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Things done
- [X] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -27338,4 +27338,42 @@ in modules // {
...
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/t/tvdb_api/${name}.tar.gz";
+      sha256 = "0hq887yb3rwc0rcw32lh7xdkk9bbrqy274aspzqkd6f7dyhp73ih";
+    };
+
+    meta = {
+      description = "Simple to use TVDB (thetvdb.com) API in Python.";
+      homepage = "https://github.com/dbr/tvdb_api";
+      license = licenses.mit;
```
Comment (groxxda) [inline pkgs/top-level/python-packages.nix]: shouldn't this be licenses.unlicense?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -27338,4 +27338,42 @@ in modules // {
...
+    };
+
+    propagatedBuildInputs = with self; [
+      tvdb_api
+    ];
+
+    meta = {
+      description = "Automatic TV episode file renamer, uses data from thetvdb.com via tvdb_api.";
+      homepage = "https://github.com/dbr/tvnamer";
+      license = licenses.mit;
```
Comment (groxxda) [inline pkgs/top-level/python-packages.nix]: same here

**Comment by peterhoeg** [comment]
Indeed @groxxda, thanks!

**Comment by peterhoeg** [comment]
Should be good to go now @groxxda

**Pull request merged**